sms_sponsorship/webapp/src/App.js

```diff
@@ -99,81 +95,80 @@ class Main extends React.Component {
                 <div>
                     {topAppBar}
                     <div>
-                        <Message text={t("error_noRequestID")}/>
+                        <Error appContext={this} text="Error : no SMS request ID. Please send an other SMS."/>
                     </div>
                 </div>
             )
         }
 
-        if (!child.has_a_child && !this.state.success) {
+        if (!child.has_a_child) {
             clearTimeout(window.getChildTimeout);
-            window.getChildTimeout = setTimeout(() => {
+            if(this.count_try > 20){
+                return (
+                            <div>
+                                <Error appContext={this} text="Error : No child found. Try again later"/>
```

Comment:
Text should be placed in the translations file and loaded with the translate function.